doc/library/config.txt

```diff
@@ -416,32 +416,36 @@ import theano and print the config variable, as in:
     `amdlibm <http://developer.amd.com/cpu/libraries/libm/>`__
     library, which is faster than the standard libm.
 
-.. attribute:: config.lib.cnmem
+.. attribute:: config.gpuarray.preallocate
 
-    Float value: >= 0
+    .. note::
```

Comment:
Move this note to the end with the other notes, we don't want to emphasize the old backend too much.